app/controllers/announcements_controller.rb

```diff
@@ -43,8 +45,8 @@ def is_english_locale?
     params[:locale].nil?
   end
 
-  def redirect_to_news_and_communications
-    base_path = "#{Plek.new.website_root}/search/news-and-communications"
+  def redirect_to_news_and_communications(atom = "")
```

Comment:
maybe 'atom' isn't such a great name for this, because it would work if you called it with 'json' for instance.

Reply:
Changing it to `format` so it's a more general parameter name